### ISSUE TITLE
Installer.Sunible.Com not capturing data

### ISSUE THREAD

**Issue by dhanur** [dhanur/Sunible-Beta#57]
"Error sending installer registration data". This page was coded by John, not Roman.

Questions, 
- Should we avoid passing data to Salesforce and use a simpler Google Doc solution for capture?
- Should we redo the page? Lots of CSS/formatting/alignment errors.

Internet Explorer
![image](https://f.cloud.github.com/assets/4460231/882308/6d8c7ca0-f960-11e2-85ef-5a629b01e9c5.png)

Chrome on Mac
![image](https://f.cloud.github.com/assets/4460231/882364/70f456c8-f961-11e2-9253-515c97a514f1.png)

**Comment by swenson86** [comment]
I think John said he was going to fix this but didn't. Maybe Google Docs is
a better option since there won't be so much data that we can key in
information into Salesforce if we need to.

**Comment by dhanur** [comment]
Yep - Google Docs is worth exploring, not just for installer but for homeowner registration as well.

 Also I'm sure Salesforce can import from Google Docs.

On July 30, 2013 at 3:06:47 PM, Tanner (notifications@github.com) wrote:

I think John said he was going to fix this but didn't. Maybe Google Docs is  
a better option since there won't be so much data that we can key in  
information into Salesforce if we need to.
